src/common/common.ts

```diff
@@ -396,6 +403,362 @@ export async function getVerifiableCredentialApiConfiguration(
   };
 }
 
+/**
+ * @param response Takes a response from a VC service and checks that it has the correct status and content type
+ * @param vcUrl The URL of the VC
+ * @returns The input response
+ */
+function validateVcResponse(
+  response: globalThis.Response,
+  vcUrl: string
+): globalThis.Response {
+  if (!response.ok) {
+    throw new Error(
+      `Fetching the Verifiable Credential [${vcUrl}] failed: ${response.status} ${response.statusText}`
+    );
+  }
+
+  const contentType = response.headers.get("Content-Type");
+
+  // Ignoring since the test suite always adds a content type
+  /* istanbul ignore if */
```

Comment:
Isn't it possible to override this in the mocked calls?

Reply:
I've added a test which does cover this